tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            pressure_vals[domain_sides.east] = 5 * self.units.Pa
</pre><p>Please use self.fluid.convert_units(5, "Pa"), which divides by the Pascal value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>